docs/source/contributing/roadmap.md

```diff
@@ -0,0 +1,86 @@
+# The repo2docker roadmap
+
+This roadmap collects "next steps" for repo2docker. It is about creating a
+shared understanding of the project's vision and direction amongst
+the community of users, contributors, and maintainers.
+The goal is to communicate priorities and upcoming release plans.
+It is not a aimed at limiting contributions to what is listed here.
+
+
+## Sharing Feedback on the Roadmap
+
+All of the community is encouraged to provide feedback as well as share new
+ideas with the community. Please do so by submitting an issue. If you want to
+have an informal conversation first use one of the other communication channels.
+After submitting the issue, others from the community will probably
+respond with questions or comments they have to clarify the issue. The
+maintainers will help identify what a good next step is for the issue.
+
+
+## What do we mean by "next step"?
+
+When submitting an issue, think about what "next step" category best describes
+your issue:
+
+* **now**, concrete/actionable step that is ready for someone to start work on.
+These might be items that have a link to an issue or more abstract like
+"decrease typos and dead links in the documentation"
+* **soon**, less concrete/actionable step that is going to happen soon,
+discussions around the topic are coming close to an end at which point it can
+move into the "now" category
+* **later**, abstract ideas or tasks, need a lot of discussion or
+experimentation to shape the idea so that it can be executed. Can also
+contain concrete/actionable steps that have been postponed on purpose
+(these are steps that could be in "now" but the decision was taken to work on
+them later)
+
+
+## Reviewing and Updating the Roadmap
+
+The roadmap will get updated as time passes (next review by 1st December) based
+on discussions and ideas captured as issues.
+This means this list should not be exhaustive, it should only represent
+the "top of the stack" of ideas. It should
+not function as a wish list, collection of feature requests or todo list.
+For those please create a
+[new issue](https://github.com/jupyter/repo2docker/issues/new).
+
+The roadmap should give the reader an idea of what is happening next, what needs
+input and discussion before it can happen and what has been postponed.
+
+
+## Project vision
+
+Repo2docker is a dependable tool used by humans that reduces the complexity of
+creating the environment in which a piece of software can be executed.
```

Comment:
> The more I think about it the more it is about sending a signal to outsiders, telling people what the values of the project are (e.g. "don't break users" vs "iteration speed!!")

@betatim I think this is a good place for the values

Reply:
I don't understand what I should add/remove from this line :-/

Reply:
I think the ones you listed before were good, stuff like "we prioritize community participation and collective decision-making over 'most fast and break stuff' development cycles".

How about:

"repo2docker run as an open and inclusive project. It encourages collective input and participation from community members, and tries to create an inviting and welcoming environment."

and we can iterate on improving this in subsequent PRs?

Reply:
@betatim Sorry I wasn't clear. I was trying to communicate (though not effectively) that you add the wording that you had suggested in a different spot here 😞 

As development and priorities require tradeoffs, we strive to preserve "backward compatibility" to not break users' usage over a "move fast - break stuff" approach.

Yes we can iterate in a subsequent PR. 👍